.github/pull_request_template.md

```diff
@@ -0,0 +1,29 @@
+## Description
+
+<!-- Describe the changes you've made. You may include any justification you want here. -->
+
+## Test Plan
+
+<!-- Explain how you tested your changes, and include any code that you used to test this. -->
+<!-- If there is an example that is sufficient to use in place of a playground, replace the playground section with a note that indicates this.  -->
+
+<details>
+
+<summary>Playground</summary>
+
+```rust
+PASTE YOUR PLAYGROUND CODE HERE
+```
+
+</details>
+
+<!-- You need to include steps regardless of whether or not you are using a playground. -->
+Steps:
+1.
+
+- [ ] I have reviewed CONTRIBUTING.md and the PR is aligned with it.
+- [ ] I have added tests for my changes.
```

Comment:
I don't think we should have these as checkboxes because it gets displayed as "tasks" when looking at the list of PRs. Perhaps a simple "Make sure you have reviewed [CONTRIBUTING.md](https://github.com/valence-rs/valence/blob/main/CONTRIBUTING.md) and there are tests to cover your changes." is enough.

Reply:
I mean they _are_ tasks that need to be done though. It forces the contributor to at least acknowledge the project's preferences existence, and helps reduce going back and forth with reviews.

Reply:
That's true, but I'm a bit worried it will clutter up the "tasks" counter especially in the cases where these things are already done before the PR is submitted. Feel free to bring this up again if you still disagree.